config/config.go

```diff
@@ -30,6 +30,7 @@ type Config struct {
 	NumTailWorkers       int              `help:"Number of tail workers to run" default:"4"`
 	NumBroadcastWorkers  int              `help:"Number of broadcast workers to run" default:"4"`
 	DemoMode             bool             `help:"Run server in demo mode. This disables modifications" default:"false"`
+	TelemetryEnabled     bool             `help:"Enable sending usage telemetry to Streamdal" default:"true"`
```

Comment:
Should be opt-in by default so `TelemetryDisabled`.

Reply:
Done!